src/backend/gl/src/command.rs

```diff
@@ -682,6 +730,8 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
                 self.cache.error_state = true;
             }
         }
+
+        self.bind_attributes();
```

Comment:
shouldn't this come before the actual draw commands?

Reply:
Yes, good catch. I missed the order when I moved this :sweat_smile: